resources/views/components/input/select/region.blade.php

```diff
@@ -0,0 +1,21 @@
+@props(['country' => ''])
```

Comment:
I think this is a required prop?

```suggestion
@props(['country'])
```

Having an empty `v-if` doesn't make any sense.